en/SetupView/Joystick.md

```diff
@@ -50,6 +50,25 @@ The expo slider allows you to make the sticks less sensitive in the center, allo
 
 The higher the Expo value, the flatter the curve is at the center, and steeper it is at the edges.
 
+## Advanced Settings
+
+![Joystick setup - Expo Curve](../../assets/setup/joystick_advanced.jpg)
+
+The advanced settings are usually not recommended for everyday users, as they can cause unpredicted results if changed wrongly. The following settings are available:
+
+- **Joystick Mode**: Changes what the joystick actually controls, and the mavlink messages sent to the vehicle.
+    - **Normal**: User controls as if using a regular RC radio, mavlink [MANUAL_CONTROL](https://mavlink.io/en/messages/common.html#MANUAL_CONTROL) messages are used.
+    - **Attitude**: User controls the vehicle attitude, mavlink [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET) messages are used.
+    - **Position**: User controls the vehicle position, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **position** only are used.
+    - **Force**: User controls the forces applied to the vehicle, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **force** only are used.
+    - **Velocity**: User controls the forces applied to the vehicle, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **velocity** only are used.
+
+- **Message Frequency**: This handles how often the joystick messages are sent. The default is 25 Hz, automatically lowered to 5Hz when there are no changes.
+
+- **Enable Circle Correction**: Circle correction is used because joysticks usually don't have as much stick travel as RC radios, these can describe a square, while joysticks usually describe a circle. This options inscribes a square into the joystick movement area. This way it is possible to reach all four corners, but this costs some resolution as the effective stick travel is reduced.
+
+- **Deadbands**: Deadbands avoid that noise or small oscilations of the sticks are interpreted as commands. They can be adjusted in the first step of calibration, or by dragging vertically on the corresponding axis monitor.
```

Comment:
Deadbands allow input changes to be ignored when the sticks are near their neutral positions. This helps to avoid noise or small oscilations on sensitive sticks are interpreted as commands, or small offsets when sticks do not re-center well. They can be adjusted during the first step of the [calibration](hyperlink), or by dragging the vertical slider on the corresponding axis monitor.

Reply:
> dragging the vertical slider on the corresponding axis monitor.

I can't get this to work, don't know how it's supposed to, keep what you have (match the comment next to enable checkbox)

Reply:
> > dragging the vertical slider on the corresponding axis monitor.
> 
> I can't get this to work, don't know how it's supposed to, keep what you have (match the comment next to enable checkbox)

This is an issue with how tall the page is. It is tall enough so it enabled scrolling, so if you try to drag the mouse up or down it scrolls instead of changing the property.

I'll open an issue about that and maybe change it so it uses horizontal movement instead.

Reply:
> or by dragging the vertical slider on the corresponding axis monitor.
There is actually no vertical slider, you were supposed to drag vertically the horizontal bar. It is weird.

Reply:
I opened an issue https://github.com/mavlink/qgroundcontrol/issues/7214, I'll probably change the logic to drag horizontally instead, makes more sense to me. Or maybe the mouse wheel?

Reply:
hmm ok I'll think more when I can try it again, ty

Reply:
I kept "or by dragging vertically on the corresponding axis monitor." since it is what the inferface says, at least until that is changed to work by dragging horizontally.